stripe.go

```diff
@@ -35,6 +35,13 @@ const (
 	// APIURL is the URL of the API service backend.
 	APIURL string = "https://api.stripe.com"
 
+	// CONNECTURL is the URL for OAuth.
+	CONNECTURL string = "https://connect.stripe.com"
```

Comment:
Haha, so admittedly this is not obvious at all. But I think `APIURL` was actually trying to follow normal naming conventions, but just ended up awkward because both parts were an acronym. (Maybe it should've been `ApiURL` instead in this case? Not sure, but whatever.)

So in this case let's go with `ConnectURL` here instead.

Reply:
When reviewing this, I don't think this is actually used anywhere and would be a convenient reference for library users, but probably not that valuable. Think I should just kill it?